src/pr-info.ts

```diff
@@ -221,7 +219,7 @@ export async function deriveStateForPR(
     // apparently `headCommit.pushedDate` can be null in some cases (see #48708), use the PR creation time for that
     // (it would be bad to use `committedDate`/`authoredDate`, since these can be set to arbitrary values)
     const lastPushDate = new Date(headCommit.pushedDate || prInfo.createdAt);
-    const lastCommentDate = getLastCommentishActivityDate(prInfo.timelineItems, prInfo.reviews) || lastPushDate;
+    const lastCommentDate = getLastCommentishActivityDate(prInfo);
```

Comment:
`lastCommentDate` is only used in https://github.com/DefinitelyTyped/dt-mergebot/blob/756610c890cbb5b381fa2a1ad501ba418da48f07/src/pr-info.ts#L241 which already includes `lastPushDate`, so ` || lastPushDate` does nothing.

Reply:
I'm kind of conflicted about it, since it might be used for other things too -- but since the type indicates the missing date possibility I guess it's fine.